woocommerce/woocommerce-ios
# [Hazmat Shipping] Add hazmat instructions links

Closes #10583

Why
==========
Inside the Hazmat section, we need to describe to the user instructions related to when a package should be declared as hazmat with links to the USPS and DHL helpers.

This PR introduces the link interaction with the instructions area of the Hazmat section, but still with no integration with models and network operations. So when we leave the package details view, the selection is lost.

How
==========
Add clickable links using markdown syntax inside the Instruction strings of the Hazmat section and taking advantage of the safari sheet to open those URLs.

Screen Capture
==========
https://github.com/woocommerce/woocommerce-ios/assets/5920403/3b21e408-3765-40dc-b768-f728c30fe252

How to Test
==========
1. Open the order details of an order eligible for Shipping label creation
2. Start the shipping label creation flow until you reach the Package details section
3. Inside the package details view, make sure the Contains hazardous materials toggle is visible and interactable
4. Make sure the toggle expands and collapses a basic view that will serve as the main control options for the HAZMAT declaration
5. Click in all three links inside the instructions area and verify they all open as a sheet inside the Package details view

Update release notes:
